src/utils/remote-state.ts

```diff
@@ -0,0 +1,154 @@
+import {DynamoDB} from "@aws-sdk/client-dynamodb";
+import {
+  CreateBucketCommand,
+  ListBucketsCommand,
+  PutBucketEncryptionCommand,
+  PutBucketVersioningCommand, PutPublicAccessBlockCommand,
+  S3Client,
+} from "@aws-sdk/client-s3";
+
+
+const ddbClient = new DynamoDB({region: "us-east-1"});
```

Comment:
it probably needs to ask for aws keys/profile? like we do in other commands

Reply:
I call it just before we run 'terraform init' , at that point user set profile already I think